src/overdue-libraries-upgrade.hook

```diff
@@ -1,11 +1,11 @@
 [Trigger]
 Operation = Upgrade
 Type = File
-Target = usr/lib/*
-Target = usr/lib64/*
```

Comment:
Well lib64 is just a symlink to lib, so its checking the same directory twice on 64-bit systems. If anyone uses the multilib repo they have lib32, so checking lib and lib32 covers everything.